plugins/slicer/MONAILabel/Resources/UI/MONAILabel.ui

```diff
@@ -645,6 +645,30 @@
         <item row="2" column="3">
          <widget class="QComboBox" name="scribLabelComboBox"/>
         </item>
+        <item row="0" column="6">
+         <layout class="QHBoxLayout" name="horizontalLayout_4">
+          <item>
+           <widget class="QLabel" name="scribblesROILabel">
+            <property name="text">
+             <string>Place ROI:</string>
```

Comment:
Lets rename this to `ROI:` 

It may be understood that this is for placing ROI...